### PR TITLE
add: `snapper-rollback-git`

### PR DESCRIPTION
**Pull request by erenfro**
Script to rollback snapper snapshots as described here https://wiki.archlinux.org/index.php/Snapper#Suggested_filesystem_layout

### CODE REVIEW

**Comment by erenfro** [comment]
This is fully tested and ready for merging.

File: packages/snapper-rollback-git/snapper-rollback-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,21 @@
+pkgname=snapper-rollback-git
+gives=snapper-rollback
+pkgver=1.0
+pkgdesc='Script to rollback snapper snapshots as described here https://wiki.archlinux.org/index.php/Snapper#Suggested_filesystem_layout'
```
Comment (Elsie19) [inline packages/snapper-rollback-git/snapper-rollback-git.pacscript:4]:
```suggestion
pkgdesc='Rollback snapper snapshots
https://wiki.archlinux.org/index.php/Snapper#Suggested_filesystem_layout'
```

File: packages/snapper-rollback-git/snapper-rollback-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+arch=('any')
```
Comment (Elsie19) [inline packages/snapper-rollback-git/snapper-rollback-git.pacscript:5]:
```suggestion
```

File: packages/snapper-rollback-git/snapper-rollback-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+makedepends=('git')
+provides=('snapper-rollback')
+backup=('etc/snapper-rollback.conf')
+install=snapper-rollback.install
+source=("https://github.com/jrabinow/snapper-rollback.git")
+sha256sums=('SKIP')
+maintainer=("Eric Renfro <psi-jack@linux-help.org>")
+
+package() {
+  cd "${_archive}"
```
Comment (Elsie19) [inline packages/snapper-rollback-git/snapper-rollback-git.pacscript:18]: Do not use _archive.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/snapper-rollback-git/snapper-rollback-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,21 @@
+pkgname=snapper-rollback-git
+gives=snapper-rollback
+pkgver=1.0
```
Comment (Elsie19) [inline packages/snapper-rollback-git/snapper-rollback-git.pacscript:3]: Quote these.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by oklopfer — APPROVED** [review]

**Pull request merged**